tools/dm/skip-replace-sqls.md

```diff
@@ -264,7 +264,7 @@ sql-replace <--worker=127.0.0.1:8262> [--binlog-pos=mysql-bin|000001.000003:3270
 
 #### Passively skip after the replication gets interrupted
 
-##### Application scenario
+##### Application scenarios
```

Comment:
It seems that we only provide one scenario here.

Reply:
PTAL again.